armi/reactor/blueprints/gridBlueprint.py

```diff
@@ -580,27 +580,32 @@ def saveToStream(stream, bluep, full=False, tryMap=False):
         if gridDesign.readFromLatticeMap or tryMap:
             geomType = geometry.GeomType.fromStr(gridDesign.geom)
             symmetry = geometry.SymmetryType.fromStr(gridDesign.symmetry)
+
```

Comment:
Why was this could move out of the try/except? 

What happens if there is an error in this code now?

Reply:
I think that there is an actual error that should be expressed and not caught with the same exception. I don't like having multiple actions in one exception because it masks the problem and it was making it harder to debug the issue.